### DISCUSSION TITLE
transform format from "deepmd" to "ABACUS"

### DISCUSSION THREAD

**Discussion by AroundPeking** [deepmodeling/dpdata#503] [Q&A]
I had a dataset in format "deepmd" like `*.npy `, which is the output of `dpgen simplify`. And I would like to transform them to a lot of abacus structure files `STRU`. How can I do it?

Thanks for your help.

**Comment by wanghan-iapcm** [comment]
you may dump data to a file of a certain format with https://docs.deepmodeling.com/projects/dpdata/en/master/#dump-data
